packages/modules/tax/README.md

```diff
@@ -1 +1,95 @@
 # Tax Module
+
```

Comment:
todo: this can be removed from that readme cc @olivermrbl

Reply:
Agree, but this is great for docs cc @shahednasser